src/dali_executor/dali_executor.h

```diff
@@ -103,6 +107,7 @@ class DaliExecutor {
   ThreadPool thread_pool_;
   std::map<std::string, IOBuffer<CPU>> cpu_buffers_;
   std::map<std::string, IOBuffer<GPU>> gpu_buffers_;
+  bool input_consumed_;
```

Comment:
Could you document, what does that flag mean? Judging from the name, it suggests, that the `DaliExecutor` can operate only on one input.

Reply:
I documented the InputsConsumed method